pallets/account-linker/src/lib.rs

```diff
@@ -114,14 +114,15 @@ decl_module! {
 			let mut addrs = Self::eth_addresses(&account);
 			// NOTE: allow linking `MAX_ETH_LINKS` eth addresses.
 			if (index >= addrs.len()) && (addrs.len() != MAX_ETH_LINKS) {
```

Comment:
Not so relevant to this pr, but here it's better to be <= than !=. Maybe currently it's a bit paranoid to do so, but later could be somewhere else addrs is pushed or size altered and here could be potentially dangerous. Same for btc.

Reply:
Watch out, it should be < instead of <=

Reply:
Besides, we really did not think about the case if it's >. Better to write this down and solve it latter